scripts/change_consts.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+mpb="DEFAULT_MILLISECS_PER_BLOCK: u64 ="
```

Comment:
I would remove the prefix `DEFAULT_` since it is not configurable other than replacing this value in the code :) 

Also there is a problem that once you run `./run_nodes.sh` then the constants in this file change, and the change is not reverted. It's not that I know a solution to this, just wanted to point out that this might be problematic ;/

Reply:
Generally I think, there should be a follow-up task to figure out how to properly build this for test scenarios... (So It might be ok to leave it as it is).